interface/main/tabs/menu/menu_data.php

```diff
@@ -117,6 +117,7 @@
       {"label":"Database","menu_id":"dba0","target":"dba","url":"/phpmyadmin/index.php","children":[],"requirement":0,"acl_req":["admin","database"],"global_req":"!disable_phpmyadmin_link"},
       {"label":"Certificates","menu_id":"adm0","target":"adm","url":"/interface/usergroup/ssl_certificates_admin.php","children":[],"requirement":0,"acl_req":["admin","users"]},
       {"label":"Native Data Loads","menu_id":"adm0","target":"adm","url":"/interface/super/load_codes.php","children":[],"requirement":0,"acl_req":["admin","super"]},
+      {"label":"Weno Management","menu_id":"adm0","target":"adm","url":"/interface/weno/admin.php","children":[],"requirement":0,"acl_req":["admin","super"]},
```

Comment:
@robertdown ,
This is just based on older code. After I finish the review, plan to rebase to most recent codebase.